demo/server/src/modules/keycloak/keycloak.d.ts

```diff
@@ -0,0 +1,4 @@
+declare var KeycloakConnect:any;
+declare module "keycloak-connect" {
+    export = KeycloakConnect;
+}
```

Comment:
:+1: Optional: we can add just a couple extra info here as the project uses it.

From what it looks like it would be something like: 
```typescript
KeycloakConnect: {
  new(options: { store: expressSession.Store }) -> KeycloakInstance
}
KeycloakInstance: {
   middleware: (options: { logout: string })
}
```